INSTALL.md

```diff
@@ -563,23 +563,21 @@ fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux",
+also known as WSL. This can be found in the "Microsoft Store" in Windows, and
+allows you to install a full copy of Ubuntu. You can then follow the guidance
+for building and using GAP in Linux inside WSL.
```

Comment:
Perhaps include a link to instructions like https://learn.microsoft.com/en-us/windows/wsl/install